### PR TITLE
Implement subtract

### PR DESCRIPTION
**Pull request by faragocsaba**

### CODE REVIEW

File: src/main/java/hu/faragocsaba/mymath/MyMath.java [hunk 1]
```diff
@@ -1,7 +1,11 @@
 package hu.faragocsaba.mymath;
 
-public class MyMath {	
+public class MyMath {
 	public static int myAdd(int a, int b) {
 		return a + b;
 	}
+
+	public static int mySubtract(int a, int b) {
+	    return a-b;
```
Comment (faragocsaba) [inline src/main/java/hu/faragocsaba/mymath/MyMath.java:9]: Please put spaces around minus sign.

Comment (farago-csaba-webuni) [inline src/main/java/hu/faragocsaba/mymath/MyMath.java]: Please add spaces around minus sign.

File: src/main/java/hu/faragocsaba/mymath/MyMath.java [hunk 2]
```diff
@@ -1,7 +1,11 @@
 package hu.faragocsaba.mymath;
 
-public class MyMath {	
+public class MyMath {
 	public static int myAdd(int a, int b) {
 		return a + b;
 	}
+
+	public static int mySubtract(int a, int b) {
```
Comment (farago-csaba-webuni) [inline src/main/java/hu/faragocsaba/mymath/MyMath.java:8]: Please add unit test.

**Review by farago-csaba-webuni — CHANGES_REQUESTED** [review]

File: src/main/java/hu/faragocsaba/mymath/MyMath.java (hunk 1 above)
Comment (faragocsaba) [reply src/main/java/hu/faragocsaba/mymath/MyMath.java]: Changed.

File: src/main/java/hu/faragocsaba/mymath/MyMath.java (hunk 2 above)
Comment (faragocsaba) [reply src/main/java/hu/faragocsaba/mymath/MyMath.java:8]: Added.

**Review by farago-csaba-webuni — APPROVED** [review]

**Pull request merged**